docs/user-guide/application-context.md

```diff
@@ -0,0 +1,122 @@
+# Application Context
+*******************
+
```

Comment:
Is there any "introduction" (here  or elsewhere) of what an Application Context is and why an admin should "desire" to create one?

Reply:
https://elena-mapstore-docs.readthedocs.io/en/5060_georchestra_review/user-guide/managing-contexts/